docs/guides/integrating-optimize.md

```diff
@@ -0,0 +1,121 @@
+---
+id: integrating-optimize
+title: Integrating Optimize
+sidebar_label: Integrating Optimize
+---
+
+<span class="badge badge--beginner">Beginner</span>
+<span class="badge badge--medium">Time estimate: 20 minutes</span>
+
+## Purpose
+
+The following document provides a basic end-to-end glance into Optimize and its features for new business users.
+
+Optimize offers business intelligence tooling for Camunda enterprise customers. By leveraging data collected during process execution, you can access reports, share process intelligence, analyze bottlenecks, and examine areas in business processes for improvement.
```

Comment:
Is "Camunda enterprise customers" the way we refer to Cloud customers using Optimize? Can professionals use Optimize? @HanselIdes

Reply:
I think we're differentiating between customers (paid) and users (any). We can probably drop the "enterprise" and not lose meaning

Reply:
Ok, let's do that then, I'll adjust it. Thanks!